help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,219 @@
+
+1. An example using these mappings is shown below:
+![Platform UI screenshot example showing Target mappings.](../../assets/catalog/crm/outreach/mappings.png)
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+* When performing the [Schedule segment export](../../ui/activate/activate-segment-streaming-destinations.md) step you must manually map Platform segments to the custom field attribute in [!DNL Outreach].
+
+* To do this, select each segment, then enter the corresponding numeric value which corresponds to the *Custom Field `N` Label* field from [!DNL Outreach] in the **[!UICONTROL Mapping ID]** field.
+    > [!IMPORTANT]
+    >
+    > * The numeric value *(`N`)* used within the Mapping ID should match the custom attribute key suffixed with the numeric value i.e. *Custom Field `N` Label* within [!DNL Outreach]. 
+    > * Note you only need to specify the number.
+    > * A maximum of *Custom Field `1` Label* ... *Custom Field `150` Label* fields are supported within [!DNL Outreach].
+    > * Refer to [[!DNL Outreach] prospect documentation](https://api.outreach.io/api/v2/docs#prospect) for details.
+
+* For example:
+    |[!DNL Outreach] Field|Platform Mapping ID|
+    |---|---|
+    | Custom Field `4` Label | `4` |
+
+    ![Platform UI screenshot showing an example Mapping Id during Schedule segment export.](../../assets/catalog/crm/outreach/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** to navigate to the list of destinations.
+![Platform UI screenshot showing Browse Destinations.](../../assets/catalog/crm/outreach/browse-destinations.png)
+
+1. Select the destination and validate that the status is **[!UICONTROL enabled]**.
+![Platform UI screenshot showing Destinations Dataflow Run for the selected destination.](../../assets/catalog/crm/outreach/destination-dataflow-run.png)
+
+1. Switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Platform UI screenshot showing Destinations Activation data.](../../assets/catalog/crm/outreach/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count created within the segment.
+![Platform UI screenshot showing Segment summary.](../../assets/catalog/crm/outreach/segment.png)
+
+1. Log in to the [!DNL Outreach] website, then navigate to the **[!DNL Apps]** > **[!DNL Contacts]** page and check if the profiles from the segment have been added. You will also notice the segment status from Experience Platform has been updated against the corresponding custom field attribute *(identified through the friendly label name)* that was provided in the Platform **Mapping ID** field during the **[!UICONTROL Activate destination]** > **[!UICONTROL Schedule segment export]** step.
```

Comment:
Line 196: I think I already mentioned this in my previous comments. There is no [!UICONTROL Activate destination] UI element. Please rephrase this section as below:

"You can see that each segment status from [!DNL Outreach] was updated with the corresponding segment status from Platform, based on the [!UICONTROL Mapping ID] value provided during the segment scheduling step."